hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/fileContext/ITestS3AFileContext.java

```diff
@@ -13,11 +13,31 @@
  */
 package org.apache.hadoop.fs.s3a.fileContext;
 
-import org.apache.hadoop.fs.TestFileContext;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.*;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
```

Comment:
move these to the top, ahead of the other imports. you may want to check your ide settings here, to avoid so much pain